examples/sn-gatsby-example/gatsby-config.js

```diff
@@ -0,0 +1,58 @@
+module.exports = {
+  siteMetadata: {
+    title: 'sn-gatsby-example',
+    description: 'A blog app with gatsby and typescript',
+    keywords: 'gatsby, javascript, sensenet, typescript',
+    siteUrl: 'https://github.com/SenseNet/sn-client.com',
+    author: {
+      name: 'Sense/Net',
+      url: 'http://sensenet.com/',
+      email: 'snteam@sensenet.com',
+    },
+  },
+  plugins: [
+    {
+      resolve: 'gatsby-source-filesystem',
+      options: {
+        name: 'content',
+        path: `${__dirname}/src/content`,
+      },
+    },
+    {
+      resolve: 'gatsby-transformer-remark',
+      options: {
+        plugins: [
+          {
+            resolve: 'gatsby-remark-responsive-iframe',
+            options: {
+              wrapperStyle: 'margin-bottom: 1rem',
+            },
+          },
+          'gatsby-remark-prismjs',
+          'gatsby-remark-copy-linked-files',
+          'gatsby-remark-smartypants',
+          {
+            resolve: 'gatsby-remark-images',
+            options: {
+              maxWidth: 1140,
+              quality: 90,
+              linkImagesToOriginal: false,
+            },
+          },
+        ],
+      },
+    },
+    'gatsby-transformer-json',
+    {
+      resolve: 'gatsby-plugin-canonical-urls',
+      options: {
+        siteUrl: 'https://gatsby-starter-typescript-plus.netlify.com',
+      },
+    },
+    'gatsby-plugin-typescript',
+    'gatsby-plugin-sharp',
+    'gatsby-transformer-sharp',
```

Comment:
https://www.gatsbyjs.com/docs/how-to/images-and-media/using-gatsby-plugin-image/

right now you don't use it in the project

Reply:
'gatsby-plugin-sharp', is needed the other one has been removed
![image](https://user-images.githubusercontent.com/22347393/113869580-2d4da180-97b1-11eb-9918-f76472cbff04.png)